doc/guides/using-symbols.md

```diff
@@ -0,0 +1,61 @@
+# Using global symbols
+
+ES6 introduced a new type: `Symbol`. This new type is _immutable_, and
+it is often used for metaprogramming purposes, as they can be used as
+property keys like string. There are two types of
+symbols, local and globals. Symbols are not included in
```

Comment:
Symbols in the GlobalSymbolRegistry have no intrinsic differences from symbols created through `Symbol()` -- other than that it is in a global map. There are no two "types" of symbols.

Reply:
From a user point of view, those are two different things. It might well be the same type, but a) I cannot add a symbol that I created to the global map and b) I cannot remove a symbol from the global map. Specifically `Symbol('hello') !== Symbol('hello')` and `Symbol.for('hello') === Symbol.for('hello')`: this alone makes me think that they are two different things. Would you mind to suggest some different wording?